frontend/src/components/custom-tools/document-viewer/DocumentViewer.jsx

```diff
@@ -30,7 +30,7 @@ function DocumentViewer({
     return (
       <div className="height-100 display-flex-center display-flex-align-center">
         <Typography.Text>
-          {errMsg || "Failed to load the document"}
+          {errMsg || "Loading onboarding document"}
```

Comment:
Will the failure cases display correctly as "Failed" @kirtimanmishrazipstack? I hope that also will not be shown as "Loading" with this change